<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm</class>
 <widget class="QMainWindow" name="ihm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bee-honey't</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="barre_laterale" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_ruches">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Ruches</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme=":/ruches.png">
                <normaloff>.</normaloff>
                <normalon>:/ruches.png</normalon>.</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_mesures">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_tableaux">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_graphiques">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme=":/graphiques.png">
                <normaloff>.</normaloff>
                <normalon>:/graphiques.png</normalon>.</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_alertes">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>:/alertes.png</normalon>
               </iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="ligne_verticale">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="ligne_horizontale_version">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_version">
              <property name="text">
               <string>v0.1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>pushButton_ruches</zorder>
           <zorder>pushButton_tableaux</zorder>
           <zorder>pushButton_mesures</zorder>
           <zorder>ligne_verticale</zorder>
           <zorder>label_version</zorder>
           <zorder>pushButton_graphiques</zorder>
           <zorder>ligne_horizontale_version</zorder>
           <zorder>pushButton_alertes</zorder>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="ruches">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QComboBox" name="comboBox_liste_ruches"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_nouvelle_ruche">
               <property name="text">
                <string>Nouvelle ruche</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_supprimer_ruche">
               <property name="text">
                <string>Supprimer ruche</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_reglage">
               <property name="text">
                <string>Réglages de la ruche</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="maximumSize">
                    <size>
                     <width>1677215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Température intérieure</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>34</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>°C</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber_2"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QComboBox" name="comboBox_donnees_affiche">
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="currentText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>800</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_rafraichir">
                 <property name="text">
                  <string>Rafraichir</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="1">
                <widget class="QChartView" name="graphicsView_5"/>
               </item>
               <item row="0" column="2">
                <widget class="QChartView" name="graphicsView_7"/>
               </item>
               <item row="1" column="2">
                <widget class="QChartView" name="graphicsView_8"/>
               </item>
               <item row="0" column="1">
                <widget class="QChartView" name="graphicsView_4"/>
               </item>
               <item row="0" column="0">
                <widget class="QChartView" name="graphicsView"/>
               </item>
               <item row="1" column="0">
                <widget class="QChartView" name="graphicsView_2"/>
               </item>
               <item row="2" column="1">
                <widget class="QChartView" name="graphicsView_6"/>
               </item>
               <item row="2" column="0">
                <widget class="QChartView" name="graphicsView_3"/>
               </item>
               <item row="2" column="2">
                <widget class="QChartView" name="graphicsView_9"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Page 3</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="graph">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QComboBox" name="comboBox_reglages_graphiques">
             <property name="maximumSize">
              <size>
               <width>205</width>
               <height>25</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Températures</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QChartView" name="chartView_temperature"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Humidité</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QChartView" name="chartView_humidite"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Luminosité</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QChartView" name="chartView_luminosite"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Pression</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QChartView" name="chartView_pression"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_5">
              <attribute name="title">
               <string>Poids</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QChartView" name="chartView_poids"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_6">
              <attribute name="title">
               <string>Activité</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QChartView" name="chartView_activite"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="alertes">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Dernières alertes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_alertes"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
